pyproject.toml

```diff
@@ -1,4 +1,116 @@
+[build-system]
+requires = ["setuptools>=61.0", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "pingouin"
+description = "Pingouin: statistical package for Python"
+authors = [
+    {name = "Raphael Vallat", email = "raphaelvallat9@gmail.com"},
+]
+dependencies = [
+    "matplotlib",
+    "numpy",
+    "pandas>=1.5",
+    "pandas_flavor",
+    "scikit-learn>=1.2",
+    "scipy",
+    "seaborn",
+    "statsmodels",
+    "tabulate",
+]
+requires-python = ">=3.8"
+readme = "README.rst"
+license = {text = "GPL-3.0"}
+maintainers = [
+    {name = "Raphael Vallat", email = "raphaelvallat9@gmail.com"},
+]
+dynamic = ["version"]
+classifiers = [
+    "Intended Audience :: Science/Research",
+    "Operating System :: MacOS",
+    "Operating System :: POSIX",
+    "Operating System :: Unix",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Topic :: Scientific/Engineering :: Mathematics",
+]
+
+[project.optional-dependencies]
+test = [
+    "pytest>=6",
+    "pytest-cov",
+    "codecov",
+    "openpyxl",
+    "mpmath",
+]
+
+[project.urls]
+Homepage = "https://pingouin-stats.org/index.html"
+Downloads = "https://github.com/raphaelvallat/pingouin/"
+
+[tool.setuptools]
+py-modules = ["pingouin"]
+include-package-data = true
+
+[tool.setuptools.package-data]
+pingouin = [
+    "datasets/*.csv",
```

Comment:
New path is `src/pingouin/datasets`

Also can you please update the datasets paths in `MANIFEST.in`?